### DISCUSSION TITLE
Sync brights between main and additional monitor doesn't work, after sleep or restart.

### DISCUSSION THREAD

**Discussion by lutogin** [MonitorControl/MonitorControl#1370] [General]
Sync brights between main and additional monitor doesn't work, after sleep or restart.
MacBook Pro 2015 macOS 11.7.3 (20G1116) + Dell U2722D
and 
MacBook Pro 2019 latest macOS + Dell U2722D

**Comment by archonium-github** [comment]
I have a similar issue.

When I boot up, external monitor goes to 100% brightness.
However waking from sleep for me, the synced settings remained.

I have Preferences setting 'Upon startup or wake' set to 'Assume last save settings are valid (recommended)'

Version 4.1.0 Build 7034
Mac OS Ventura 13.3.1 (a) M1 Max 2022 MacBook Pro

**Comment by lutogin** [comment]
More details:
As I understood, additional monitor after sleep or restart lose sync with main monitor.
If in app, press Reset to default for main monitor, sync starts working.
Also, after sleep or restart, if bright is changed due to the daylight, or I change it by hotkeys on keyboard, bright level changes but in app, it still without any changes

I tried all modes in Upon startup or wake - it doesn't work.

Please help with that. It's really annoy

**Comment by archonium-github** [comment]
After installing the first time and then restarting over a few days Monitor Control didn't remember my external monitors screen brightness settings. 

The first time, it set my external monitor to 100% brightness. I just use the menu bar to reset the brightness to its appropriate level. (I use separate sliders as there is a difference in brightness between my MacBook [33%] and external monitor [48%]).

At some point Monitor Control started working and remembers my settings even after restarting/rebooting.

Occasionally it would restart with the percentage on my external monitor changing slightly  (from my setting of 48% it would sometimes start up with 50%; doesn't bug me, I just reset it)

In general it's working fine for me. Love it!

MacBook Pro M1 Max Ventura 14.5
LG Ultragear 27GN60R-B
